exec/java-exec/src/main/java/org/apache/drill/exec/ops/ContextInformation.java

```diff
@@ -63,4 +65,11 @@ public long getQueryStartTime() {
   public int getRootFragmentTimeZone() {
     return rootFragmentTimeZone;
   }
+
+  /**
+   * @return HLL accuracy parameter
+   */
+  public int getHllMemoryLimit() {
```

Comment:
The comment says this is an accuracy parameter, but the method name says it is a memory limit. Perhaps explain how the two relate?